### ISSUE TITLE
Revamp Application

### ISSUE THREAD

**Issue by haodev007** [parasol-finance/application#179]
- [ ] Eliminate unused, unnecessary and duplicated code(DRY).
- [ ] Uninstall unused packages
- [ ] Use of duplicated tailwind utility classes and remove.
- [ ] Resolve of all responsive issues on all devices
- [ ] Implement extensive app structure
- [ ] Create `src` Directory and alternative to the root `pages` (baseUrl options)
- [ ] Code splitting to reduce the size of app's first payload
- [ ] Move all temp data of `components` folder and `pages`  folder to `utils` folder
- [ ] Move all types and interfaces for `components` folder and `pages` folder to `types` folder
- [ ] Import order(Built-in, External, Internal) and Implement of dynamic import
- [ ] Create context for store management
- [ ] Create reusable components and Use it properly
- [ ] Create Custom Document(`_document` file)
- [ ] Usage of image assets in public directory and import rule
- [ ] Customize of Next Image(blur effect while loading and more)
- [ ] Caching of all assets for optimization
- [ ] Automatic Font Optimization(Add font to Document)
- [ ] Implement Server Side Rendering(SSR) - getServerSideProps()
- [ ] Data fetching optimization using SWR
- [ ] Babel to compile server-side code and custom config
- [ ] Switchable Next.js Runtime (https://github.com/vercel/next.js/discussions/34179)

**Comment by haodev007** [comment]
Yeah, we will need and do it `Body text color https://tailwindcss.com/docs/adding-custom-styles#adding-base-styles` in #4
I think we will figure them out in this issue (`Uniformize styles, Uniformize components, use utils folders and reduce number of files`)
But I am not sure what I can do with these parts
`https://raw.githubusercontent.com/solana-labs/token-list/main/assets/mainnet/EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v/logo.png`
`Wallet connect + KYC`

**Comment by clintnetwork** [comment]
Wallet connect lets forget.

**Comment by haodev007** [comment]
I have added fix list of validate DOM nesting bug into this issue. I think this is just rule of HTML markup use.
![image](https://user-images.githubusercontent.com/98631674/171879343-8c2e1fb0-07ed-42cb-919f-88086ce552fb.png)
![image](https://user-images.githubusercontent.com/98631674/171879359-67d96788-841c-4341-8c4a-f658861cd737.png)
